content/2017.03.building-projects-as-a-distributed-team.rst

```diff
@@ -0,0 +1,178 @@
+=================================================
+
+Working in a remote team is really similar to working with a
+community.
+
+You need to explain everything you do, you need to find consensus,
+you need to take feedback, you need to propose solutions and try
+things.
+
+As a team, as a community, you are stronger because instead of being
+alone you can rely on others to help you make the best choices and
+write the best software.
+
+
+The Prototype
+=============
+
+You've got an idea, let's make it a prototype.
+
+Find out what are the essential features that would solve the problem.
+
+Write a document (article, blue print) to explain what you decided to
+build as a prototype.
+
+At this stage it is really important to get something that looks close
+enough to the minimum feature set we want, but it should be really
+simple. In two days you should be able to demo the UX of your
+prototype and in two weeks something working well enough.
+
+It is really important to do that because it will bring a lot of
+questions, you will realize that the prototype is different from each
+thoughts of how it should look likes.
+
+This is especially true if you are working with people from differents
+jobs that are not looking at the same things. (Backend developers, UX
+designers, users, ...)
+
+The faster you get your prototype ready the faster you can align on
+what you all want for your MVP.
+
+
+The Minimal Viable Product (MVP)
+================================
+
+Now that we have a prototype, the question now is what is missing to
+make a product.
+
+It is time to gather together and brainstorm around the
+project to define what is the **Minimum Viable Product**.
+
+At this stage it is really important to start talking of a product,
+the project version ``1.0.0`` starts with a ``1`` which means that you
+are trying to put together the first actual version of your product.
+
+Everything that is not in the feature set of the MVP is put in a
+backlog for later.
+
+It is important to document the fact that we thought about it and that
+we will possibly care about it later.
+
+At each stage — Prototype, MVP — we write a small article to summarize
+the discussion and to explain what is the feature set, what is the
+protocol.
+
+You might think that writing an article about it should be optional,
+but you will realize that if you don't do it, you won't remember on
+what you'd agree and people of your team will start asking questions
+about why you did it like that. If the expectation change along the
+way we should notice it and take decision to update the document.
+
+
+Answering complex questions
+---------------------------
+
+Some features of the MVP might not be trivial. Some decision are hard
+to make between two solutions.
+
+You will have to carefully explain pros and cons and why you choose to
+do it like that. If it is written it is easy to explain people who
+forgot about it why you did it like that.
```

Comment:
`why you choose to do it like that` is a little weird. Maybe `why you made the choice you did`? Comma after `written`, `explain to people`. Instead of `forgot`, how about `explain to people who don't remember what happened why you did it like that`? (Although that's kind of clumsy.)